[INFO   ] Logger: Record log in C:\Users\si\.kivy\logs\kivy_22-06-09_22.txt
[INFO   ] deps: Successfully imported "kivy_deps.angle" 0.3.2
[INFO   ] deps: Successfully imported "kivy_deps.glew" 0.3.1
[INFO   ] deps: Successfully imported "kivy_deps.sdl2" 0.4.5
[INFO   ] Kivy: v2.1.0
[INFO   ] Kivy: Installed at "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\__init__.py"
[INFO   ] Python: v3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)]
[INFO   ] Python: Interpreter at "C:\Users\si\AppData\Local\Programs\Python\Python310\python.exe"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 189 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.6.0 NVIDIA 512.15'>
[INFO   ] GL: OpenGL vendor <b'NVIDIA Corporation'>
[INFO   ] GL: OpenGL renderer <b'NVIDIA GeForce GTX 1660/PCIe/SSE2'>
[INFO   ] GL: OpenGL parsed version: 4, 6
[INFO   ] GL: Shading version <b'4.60 NVIDIA'>
[INFO   ] GL: Texture max size <32768>
[INFO   ] GL: Texture max units <32>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Base: Start application main loop
[INFO   ] GL: NPOT texture support is available
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "c:\Users\si\OneDrive\Documents\Desktop\Projects\Automate.py", line 43, in <module>
[WARNING] stderr:     uiApp().run()
[WARNING] stderr:   File "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\app.py", line 955, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\base.py", line 574, in runTouchApp
[WARNING] stderr:     EventLoop.mainloop()
[WARNING] stderr:   File "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\base.py", line 339, in mainloop
[WARNING] stderr:     self.idle()
[WARNING] stderr:   File "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\base.py", line 383, in idle
[WARNING] stderr:     self.dispatch_input()
[WARNING] stderr:   File "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\base.py", line 334, in dispatch_input
[WARNING] stderr:     post_dispatch_input(*pop(0))
[WARNING] stderr:   File "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\base.py", line 302, in post_dispatch_input
[WARNING] stderr:     wid.dispatch('on_touch_up', me)
[WARNING] stderr:   File "kivy\_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\uix\behaviors\button.py", line 179, in on_touch_up
[WARNING] stderr:     self.dispatch('on_release')
[WARNING] stderr:   File "kivy\_event.pyx", line 727, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "kivy\_event.pyx", line 1307, in kivy._event.EventObservers.dispatch
[WARNING] stderr:   File "kivy\_event.pyx", line 1191, in kivy._event.EventObservers._dispatch
[WARNING] stderr:   File "C:\Users\si\AppData\Local\Programs\Python\Python310\lib\site-packages\kivy\lang\builder.py", line 55, in custom_callback
[WARNING] stderr:     exec(__kvlang__.co_value, idmap)
[WARNING] stderr:   File "<string>", line 10, in <module>
[WARNING] stderr: NameError: name 'driver' is not defined
